fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/bridge/api/complex/SumoLaneChangeMode.java

```diff
@@ -51,16 +55,10 @@ public static SumoLaneChangeMode translateFromEnum(final LaneChangeMode laneChan
                 mode.setStrategicChanges(true, true);
                 mode.setCooperativeChanges(true, false);
                 mode.setSpeedChanges(true, false);
-                mode.setRightDriveChanges(true, true);
+                mode.setRightDriveChanges(true, false);
                 mode.setRespectOtherDrivers(RESPECT_SPEED_GAPS_OF_OTHER_DRIVERS_DO_NOT_ADAPT_SPEED);
                 break;
             case PASSIVE:
```

Comment:
please check passive and cooperative

Reply:
The only difference between PASSIVE and COOPERATIVE was that the second bool in `setRightDriveChanges()` was set to `true` for PASSIVE. If the second boolean is set to true, than lane-changes via TraCI(MOSAIC API) are ignored if the lane-change model itself wants to keep the right lane. Ignoring TraCI at this point is not helpful imo. Only strategic lane changes should override lane-changes via TraCI, to keep route-following alive. 

Tl;dr from my pov it's fine as it is here now.